services/frontend-service/api

```diff
@@ -1 +1 @@
-../../pkg/api
\ No newline at end of file
+../../pkg/api/v1
```

Comment:
is this correct?

Reply:
No, nice catch. That was specific to my local setup.

Reply:
Yes it is. It does not affect CI builds anyhow, it is however needed for local development. It was needed after 

The reason this change is here is because this branch was created before the fix on the main branch #1323.